src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Bulldozer is now working");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Bulldozer has stopped working");
```

Comment:
Similarly to the `doWork` method, make the stop message more informative by including the class name, such as: 'Bulldozer stopped its work.'